templates/blocks/header.twig

```diff
@@ -3,7 +3,9 @@
 	<div class="page-header">
 		{% if ( background_image ) %}
 			<div class="page-header-background">
-				<img src="{{ background_image }}" class="page-header-image" />
+				<img src="{{ background_image }}"
+					 srcset="{{ background_image_srcset }}"
```

Comment:
Please add a check here if background_image_srcset is set, in case wp_get_attachment_image_srcset returns `false`.

Reply:
Added it.